### PR TITLE
scrub controller methods

### PR DESCRIPTION
**Pull request by deads2k**
Move controller initialization closer to where they belong and further from the apiserver

### CODE REVIEW

File: pkg/cmd/server/kubernetes/master/controller/config.go [hunk 1]
```diff
@@ -45,3 +58,46 @@ func (c KubeControllerConfig) GetControllerInitializers() (map[string]kubecontro
 
 	return ret, nil
 }
+
+func BuildKubeControllerConfig(options configapi.MasterConfig) (*KubeControllerConfig, error) {
```
Comment (mfojtik) [inline pkg/cmd/server/kubernetes/master/controller/config.go]: godoc?

**Comment by deads2k** [comment]
@mfojtik updated.  otherwise?

**Comment by mfojtik** [comment]
/lgtm

nice job :)

**Pull request merged**